lambdas/build-cri-oauth-request/src/main/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandler.java

```diff
@@ -146,7 +149,10 @@ public Map<String, Object> handleRequest(JourneyRequest input, Context context)
             String ipAddress = getIpAddress(input);
             String featureSet = getFeatureSet(input);
             configService.setFeatureSet(featureSet);
-            String journey = getJourney(input);
+            URI journeyUri = URI.create(input.getJourney());
+            String journey = journeyUri.getPath();
```

Comment:
Can we name this `journeyPath` instead - given that the `journey` parameter on the requests etc. is always the full URI